frontend/scripts/react-components/explore/explore.component.jsx

```diff
@@ -25,7 +25,7 @@ import {
 
 import 'react-components/explore/explore.scss';
 
-const ToolLinksModal = React.lazy(() => import('./tool-links-modal'));
+const ToolLinksModal = React.lazy(() => import(/* webpackPreload: true */ './tool-links-modal'));
```

Comment:
https://caniuse.com/#feat=link-rel-preload

Are we sure we can use preload? Check firefox and now IE

Reply:
If it's not supported it will load it when needed, so it's ok.